addon/export.py

```diff
@@ -370,19 +370,21 @@ def export_openscenario(self):
             for obj in bpy.data.collections['OpenDRIVE'].objects:
                 if obj.name.startswith('junction'):
                     if not len(obj['incoming_roads']) == 4:
-                        self.report({'ERROR'}, 'Junction must have 4 connected roads.')
-                        break
+                        self.report({'WARNING'}, 'Junction should have 4 connected roads.')
```

Comment:
I would say move the warning down into the for loop and throw it for every missing connection.

Reply:
It is wonderful, thanks